assets/css/style.scss

```diff
@@ -2,6 +2,7 @@
 ---
 
 @import "{{ site.theme }}";
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap");
```

Comment:
why is it necessary to also import it here?

Reply:
The HTML header is just a 'preload' to optimize page load time. The `@import` is what actually applies it as CSS. `@import {{ site.theme }}` loads the remote theme, which has its own `@import` for the font, that does not include true italics, so I have to override it here.

Reply:
it's only true italics if it comes from the Italics region of CSS? Can't believe we've been living with sparkling words all this time.